releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -2,6 +2,9 @@
 Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version information can be found in the console when the task is executed:
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
+### v6.4.0 - November, 20 2023
+* Support appending prefixes to feature flags.

```

Comment:
Added support for a feature that allows users to add a prefix to the key name of feature flags. This will provide more flexibility and better organization of feature flags for users.

(Please add a reference to the relevant GitHub issue)

Reply:
Updated string as suggested. Added link to github issue